docs/markdown/Using Pants/validating-dependencies.md

```diff
@@ -128,6 +128,8 @@ The dependency `src/a/main.py` -> `src/b/lib.py` would consult the `__dependenci
 
 When declaring your rules, you not only provide the rules for the current directory, but also set the default rules for all the subdirectories as well. When overriding such default rules in a child BUILD file, there is a `extend=True` kwarg you may use if you want the default rules to still apply after the ones declared in the current BUILD file.
 
+Note: Any rule globs using the declaration path anchoring mode that is inherited using `extend=True` will be anchored to the path of the current BUILD file, not the original one where the rule was extended from.
```

Comment:
This is a little confusing to me, maybe an example would be helpful?

Reply:
Did the added example improve the situation?